crates/bevy_render/src/primitives/mod.rs

```diff
@@ -72,14 +72,26 @@ impl Sphere {
     }
 }
 
-/// A plane defined by a normal and distance value along the normal
+/// A plane defined by a normalized normal and distance value along the normal
 /// Any point p is in the plane if n.p = d
-/// For planes defining half-spaces such as for frusta, if n.p > d then p is on the positive side of the plane.
+/// For planes defining half-spaces such as for frusta, if n.p > d then p is on the positive side (inside) of the plane.
 #[derive(Clone, Copy, Debug, Default)]
 pub struct Plane {
     pub normal_d: Vec4,
```

Comment:
```suggestion
    normal_d: Vec4,
```
And then update other code that uses `Plane` to use `Plane::new` for construction.